city_scrapers/spiders/chi_board_elections.py

```diff
@@ -0,0 +1,167 @@
+        needs.
+        """
+        if response.url == \
+                "https://app.chicagoelections.com/pages/en/meeting-minutes-and-videos.aspx":
+            yield from self._prev_meetings(response)
+        if response.url == \
+                "https://app.chicagoelections.com/pages/en/board-meetings.aspx":
+            yield from self._next_meeting(response)
+
+        # self._parse_next(response) yields more responses to parse if necessary.
+        # uncomment to find a "next" url
+        # yield self._parse_next(response)
+
+    def _next_meeting(self, response):
+        next_meeting = response.xpath('//div[@class="copy"]/text()').extract()[2]
+        meetingdate = re.search(r'\d{1}:.{24}', next_meeting).group(0)
+        data = {
+            '_type': 'event',
+            'name': "Chicago Board of Election Commissioners",
+            'event_description': "Meeting",
+            'classification': COMMISSION,
+            'start': self._parse_start(meetingdate),
+            'end': {},
+            'all_day': False,
+            'location': self._parse_location(response),
+            'documents': [],
+            'sources': [{
+                'url': "https://app.chicagoelections.com/pages/en/board-meetings.aspx",
+                'note': ''
+            }],
+        }
+
+        data['status'] = self._generate_status(data)
+        data['id'] = self._generate_id(data)
+        yield data
+
+    def _prev_meetings(self, response):
+        meetings = response.xpath("//a/text()").extract()
+        meetingdates = [
+            meeting[22:]
+            for meeting in meetings
+            if "Minutes" not in meeting and "mode" not in meeting and meeting is not " "
+        ]
+        for meetingdate in meetingdates:
+            meetingdate = "9:30 a.m. on " + meetingdate
+            data = {
+                '_type': 'event',
+                'name': "Chicago Board of Election Commissioners",
+                'event_description': "Meeting",
+                'classification': COMMISSION,
+                'start': self._parse_start(meetingdate),
+                'end': {},
+                'all_day': False,
+                'location': self._parse_location(response),
+                'documents': [],
+                'sources': [{
+                    'url':
+                        "https://app.chicagoelections.com/pages/en/meeting-minutes-and-videos.aspx",
+                    'note': ''
+                }],
+            }
+
+            data['status'] = self._generate_status(data)
+            data['id'] = self._generate_id(data)
+
+            yield data
+
+    def _parse_next(self, response):
+        """
+        Get next page. You must add logic to `next_url` and
+        return a scrapy request.
+        """
+        next_url = None  # What is next URL?
+        return scrapy.Request(next_url, callback=self.parse)
+
+    def _parse_name(self, item):
```

Comment:
This, `_parse_description`, `_parse_end`, `_parse_all_day, and `_parse_sources` can be removed since they aren't being used